OSSM-11010 [DOCS] Update OSSM tracing docs for ambient mode (#102984)
openshift/openshift-docs: modules/ossm-config-otel.adoc

[id="ossm-config-otel_{context}"]
= Configuring {OTELName} with Service Mesh

[role="_abstract"]

> You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export OpenTelemetry traces, metrics, and logs to analyze and understand your software's performance and behavior.

Reviewer: Suggestion to consider:
```suggestion
You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export OpenTelemetry traces, metrics, and logs to analyze and understand the performance and behavior of the software.
```
ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

Do not use possessive ’s with inanimate objects.

.Prerequisites

* You have installed the {TempoOperator}. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/distributed_tracing/distr-tracing-tempo-installing#installing-the-tempo-operator_distr-tracing-tempo-installing[Installing the Tempo Operator].

> * You have installed the {OTELOperator}. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/red_hat_build_of_opentelemetry/install-otel[Installing the Red Hat build of OpenTelemetry]

Reviewer 2: Small nit here: missing punctuation at the end:

```suggestion
* You have installed the {OTELOperator}. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/red_hat_build_of_opentelemetry/install-otel[Installing the Red Hat build of OpenTelemetry].
```

> * You have installed a `TempoStack` which is configured in a `tempo` namespace. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/distributed_tracing/distr-tracing-tempo-installing#installing-a-tempostack-instance_distr-tracing-tempo-installing[Installing a TempoStack instance].

Author: Not followed this style elsewhere in the OSSM docs. So I have made the changes compared to how it is to maintain consistency.

----
<1> The `service` field is the `OpenTelemetry` collector service in the `istio-system` namespace.

> . Create an Istio Telemetry resource to enable tracers defined in `spec.values.meshConfig.ExtensionProviders`, similar to the following example:

Reviewer 2: Checking if the attribute should be used here:
```suggestion
. Create an {istio} Telemetry resource to enable tracers defined in `spec.values.meshConfig.ExtensionProviders`, similar to the following example:
```
